randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -343,11 +345,18 @@ Extra - room_id: [8]
               # Use Pillar to Morph Ball Jump and place a Bomb
               Can Activate Pillar
               Any of the following:
-                  Can Use Springball
-                  # Bomb Jump and place a bomb - https://youtu.be/zOmdb38qpkk
-                  Movement (Beginner) and Can Use Bombs
+                  Can Use Power Bombs
+                  All of the following:
+                      # With Bombs
+                      Can Use Bombs
+                      Any of the following:
+                          Can Use Springball
+                          # Bomb Jump and place a bomb - https://youtu.be/zOmdb38qpkk
+                          Movement (Beginner)
           # Alternate - Screw Attack and some extra Jump height (knowledge)
           Screw Attack and Knowledge (Beginner) and Can Climb High Jump Wall
+          # Path for when player doesnt know that the PB activates the pillar
+          Can Climb High Jump Wall and Can Use Power Bombs
```

Comment:
I don't think this block is needed, or at least this comment is misleading.
PBing pillars is considered trickless according to the template, which I agree with tbh.

Reply:
Same.